cookbook/bme280_environment.rst

```diff
@@ -51,7 +52,13 @@ After validating the sensor is working, we can proceed and add some formulas.
             ((273.15 + id(bme280_temperature).state) * r); // in grams/m^3
         accuracy_decimals: 2
         update_interval: 15s
+        icon: 'mdi:water'
         unit_of_measurement: 'g/m³'
+      - platform: template
+        name: "Dew Point"
+        lambda: return (243.5*(log(id(bme280_humidity).state/100)+((17.67*id(bme280_temperature).state)/(243.5+id(bme280_temperature).state)))/(17.67-log(id(bme280_humidity).state/100)-((17.67*id(bme280_temperature).state)/(243.5+id(bme280_temperature).state))));
```

Comment:
Could you add some line breaks to make it easier to the in the published docs? Thanks.

Reply:
I broke some lines :wink: